src/EnumSet.php

```diff
@@ -484,36 +690,132 @@ public function setBinaryBitsetBe($bitset)
 
     /**
      * Get a bit at the given ordinal number
-     * 
-     * @param $ordinal int Ordinal number of bit to get
+     *
+     * @param int $ordinal Ordinal number of bit to get
+     * @return boolean
+     */
+    public function getBit($ordinal)
```

Comment:
again, check for $ordinal being integer?

Reply:
It's using the same behavior as standard PHP function in PHP-5.6 and even with PHP-7.0 with strict-types=false.
-> We should discuss it in a different thread/issue